kafka/adminapi.go

```diff
@@ -859,6 +859,14 @@ func (a *AdminClient) GetMetadata(topic *string, allTopics bool, timeoutMs int)
 	return getMetadata(a, topic, allTopics, timeoutMs)
 }
 
+// GetConsumerGroupMetadata queries broker for consumer groups metadata and all its members
+// like which consumer groups and which corresponding members consume which topics
+// if consumerGroup is nil, metadata for all consumer groups will be returned
+// otherwise only metadata for that specific consumer group will be returned
+func (a *AdminClient) GetConsumerGroupMetadata(consumerGroup *string, timeoutMs int) ([]ConsumerGroupMetadata, error) {
```

Comment:
I think this should probably be called DescribeConsumerGroup() to be somewhat consistent with https://cwiki.apache.org/confluence/display/KAFKA/KIP-222+-+Add+Consumer+Group+operations+to+Admin+API